vic/drivers/shared_image/src/check_domain_info.c

```diff
@@ -58,7 +58,7 @@ compare_ncdomain_with_global_domain(char *ncfile)
     // loop over all grid cells and check that the two domains are identical
     for (i = 0; i < global_domain.ncells_total; i++) {
         // mask matches
-        if (ncfile_domain.locations[i].run > global_domain.locations[i].run) {
+        if (ncfile_domain.locations[i].run < global_domain.locations[i].run) {
```

Comment:
I don't see the function of the domain file in case you don't want to be able to turn on an off which grid cells are run.

I think it is usefull to use a file in which you define the cells to run. This "domain-file" should contain only one variable: `run_cells` or someting.

For us it would be even better to be able to use larger param/forcing files than the domain-file. Not only in number of 'run_cells' but also in extend (lat/lon dimension lengths). The advantage is that you dont need to generate new param files and crop-forcing files every time you run on a different part of the total domain. The output of VIC will then be the same as the domain-file (also the lat/lon lengths).

Apart from the discussion: can we just merge this little bug-fix for the time beeing? My test setups do not work without...